jmxtrans-utils/src/main/java/com/googlecode/jmxtrans/util/NumberUtils.java

```diff
@@ -43,4 +43,26 @@ public static boolean isNumeric(Object value) {
 		return false;
 	}
 
+	public static boolean isValidNumber(Object value) {
```

Comment:
Correct.  instanceof checks for null always return false.  I can make it explicit if you wish.